### ISSUE TITLE
Make sure all controls are associated with labels

### ISSUE THREAD

**Issue by ewinslow** [Elgg/Elgg#5712]
The following are currently missing labels:
 * [x] Username and password fields on login form
 * [x] River selector
 * [x] Category filter on admin/plugins page
 * [x] Sort dropdown on admin/plugins page

This is not an exhaustive list.

**Comment by jdalsem** [comment]
If more locations pop up we can provide new PR's / issues for those
